frontend/src/views/ElectedCanteens.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="text-left">
+    <BreadcrumbsNav />
+    <h1 class="text-h6 mt-n2 mb-2 font-weight-bold">
+      Les cantines de mon territoire
+    </h1>
+    <p>Établissements dans {{ departmentsString }}</p>
+    <v-data-table
+      :footer-props="footerProps"
+      :items="visibleCanteens"
+      :options.sync="options"
+      :loading="loading"
+      :headers="headers"
+      :server-items-length="canteenCount || 0"
+    >
+      <template v-slot:[`item.publicationStatus`]="{ item }">
+        <router-link
+          v-if="item.publicationStatus === 'published'"
+          :to="{ name: 'CanteenPage', params: { canteenUrlComponent: `${item.id}--${item.name}` } }"
+        >
+          Publiée
+        </router-link>
+        <span v-else>
+          Non publiée
+        </span>
+      </template>
```

Comment:
Maybe add a link to the contact form to give them a way of asking for updates to their list of territories somewhere on the page (either after the departmentsString or after this table?)

Reply:
Added here: https://github.com/betagouv/ma-cantine/pull/2904/commits/e09208d9e5faf5c48d3347cd230b24f0c47fbf29